searx/engines/google.py

```diff
@@ -41,7 +41,7 @@
 }
 
 # engine dependent config
-categories = ['general']
+categories = ['general', 'powered by google']
```

Comment:
I understand categories like 'repos', 'wikis', 'database' ... theses categories classify engines ... but I do not understand how 'powered by ...' fits into this scheme of classifications.

We need a documentation of the intention of subcategories, we should name what _default_ set of subcategories SearXNG has and we should give some rules at hand how to classify new added engines well in the right subcategory.